### ISSUE TITLE
design - convert JB branding vectors from Adobe formats into open formats

### ISSUE THREAD

**Issue by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#219]
We would **looooove** to have someone help convert some proprietary .ai and .indd design files into open formats (svg, etc). This would likely be a batch operation, and you can work closely with me to get access to the files.

Marking as high priority since this is coming up again and again as a barrier to progress.

**Comment by reesericci** [comment]
I was able to get all the rocket logo files over to svg! (@gerbrent has them now)

**Comment by reesericci** [comment]
The artwork still needs to be converted over though - right?

**Comment by gerbrent** [comment]
If by "artwork" you mean all of the show artwork, then... yes. We have alot of assets that could use a conversion. 
This issue will be closed once satisfied ; )

**Comment by gerbrent** [comment]
Added more specifics to the top comment via todo list..

**Comment by wilron** [comment]
@gerbrent I'm interested in trying out a couple of the conversions. It may be easiest for me to start with some .ai files.

**Comment by SoundRisk** [comment]
I would like to help with this, how can I get started? I'm also new to GIT/GITHUB fyi. Thanks!

**Comment by gerbrent** [comment]
hey @SoundRisk ! Welcome!

Lets check in w @wilron to see of they still want to continue this work. Honestly, I'm likely the bottleneck here in providing files.. if that's the case, I can kick into gear!

**Comment by gerbrent** [comment]
Yep, confirmed that @wilron is just waiting on my - I'm the bottle neck. Whoopsie!

**Comment by wilron** [comment]
I'm happy to share the effort as well. It's up to you all on how you'd like
to proceed, but I'm happy to share the effort :-)

**Comment by ozcoder** [comment]
![rocket_orig](https://user-images.githubusercontent.com/102447/192098198-2d80a595-9c53-4b8f-8b4f-ae63d1382445.svg)

Found this rocket I created a while ago.
```<svg xmlns="http://www.w3.org/2000/svg">
<path fill="#fefffe" d="M 261.5,63.5 C 262.497,63.4701 263.164,63.9701 263.5,65C 288.441,120.26 303.941,178.093 310,238.5C 310.971,268.904 310.304,299.237 308,329.5C 311.563,322.041 315.397,
314.707 319.5,307.5C 320.5,306.167 321.5,306.167 322.5,307.5C 342.816,332.49 354.649,361.157 358,393.5C 360.527,430.003 356.194,465.67 345,500.5C 344.586,500.957 344.086,501.291 343.5,501.5C
 342.077,476.73 334.577,454.064 321,433.5C 314.883,426.377 307.883,420.377 300,415.5C 297.604,433.857 293.604,451.523 288,468.5C 285.091,475.324 281.424,481.658 277,487.5C 261.876,497.465 24
9.876,494.465 241,478.5C 238.117,473.401 235.783,468.067 234,462.5C 229.56,447.073 226.227,431.406 224,415.5C 203.931,428.593 191.264,446.927 186,470.5C 184.166,480.838 182.333,491.171 180.5
,501.5C 177.985,496.791 176.151,491.791 175,486.5C 171.099,469.995 168.099,453.328 166,436.5C 160.976,390.614 171.642,348.947 198,311.5C 199.5,310 201,308.5 202.5,307C 203.332,306.312 203.99
8,306.479 204.5,307.5C 208.588,314.676 212.422,322.009 216,329.5C 213.727,299.902 213.061,270.236 214,240.5C 220.009,178.791 235.843,119.791 261.5,63.5 Z"/>
</svg>

**Comment by wilron** [comment]
Sorry, I'm going to un-assign myself on this one as I haven't been able to make any progress without the files to convert. I'll look around for another issue to contribute to.

**Comment by CGBassPlayer** [comment]
No worries @wilron! Thank you for the efforts you have provided

**Comment by SoundRisk** [comment]
@gerbrent can we get some files, please?

**Comment by gerbrent** [comment]
oh yes, I am by far the worst.

partly: best way to share files that can be organized, protected, yet high usability?

maybe git annex or git-lfs?

anyone have any experience here w these?

**Comment by gerbrent** [comment]
also: sorry @wilron ! Its not you, its me ; )

**Comment by reesericci** [comment]
@gerbrent git-lfs could work!, or possibly like a figma with everything on it

**Comment by SoundRisk** [comment]
@gerbrent git-lfs is good by me also!
